### PR TITLE
Create ui-c8y-1020-39-0-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60342](https://cumulocity.atlassian.net/browse/MTM-60342)
Original PR: [7347](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7347)

[MTM-60342]: https://cumulocity.atlassian.net/browse/MTM-60342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-39-0-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: GAINSIGHT: Add more properties to the dashboards custom event (#6872) [GRAFT][release/cd] (#7347)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-39-0-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-graft.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Gainsight: additional properties added to dashboards custom event
```

File: content/change-logs/application-enablement/ui-c8y-1020-39-0-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60342
+version: 1020.39.0
+---
+GAINSIGHT: Add more properties to the dashboards custom event (#6872) [GRAFT][release/cd] (#7347)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-39-0-gainsight-add-more-properties-to-the-dashboards-custom-event-6872-graft.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the data available for analysis in Gainsight, additional properties have been added to the custom event that is triggered when users view or create a dashboard. The custom event now includes information such as the dashboard ID, the ID of the user performing the action, and the type of action (view or create). This change will provide more detailed insights into dashboard usage without impacting the user experience, but may result in additional data being sent to Gainsight for environments that use this integration.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by rahulc8y — CHANGES_REQUESTED** [review]
I don't think we should include information about Gainsight-related changes in our customer facing changelog. After reviewing the last 1 year's changelog, I did not find similar information from other c8y products.

**Comment by DawidSAG** [comment]
> I don't think we should include information about Gainsight-related changes in our customer facing changelog. After reviewing the last 1 year's changelog, I did not find similar information from other c8y products.

Yes, true. I've missed adding label to the PR, which prevents creating it.

**Pull request closed without merging**